src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -36,12 +37,14 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
   ///
   /// \param io_service The event loop to run the monitor on.
   /// \param node_info_accessor The node info accessor.
-  /// \param error_info_accessor The error info accessor, which is used to report error
+  /// \param error_info_accessor The error info accessor, which is used to report error.
+  /// \param gcs_pub_sub GCS message pushlisher.
```

Comment:
`when detecting the death of nodes` should be after `report error`.
Can you also add a comment for `gcs_table_storage`?